setup.py

```diff
@@ -4,6 +4,8 @@
 
 setup(
     setup_requires=['pbr<=6.0.0'],
+    dependency_links=['https://bitbucket.org/atlassian/httptest/get/0.8.zip'
```

Comment:
This needs / should be available from `pypi`.

Reply:
I wasn't aware it was published to PyPi (https://pypi.org/project/httptest/ is something entirely different), but I'll switch that!